syft_tensorflow/tensor/keras_model.py

```diff
@@ -0,0 +1,104 @@
+import weakref
```

Comment:
can we rename the folder containing this file to `syft_tensorflow/types/`, ``syft_tensorflow/classes/`, `syft_tensorflow/objects/`, or something similar but more informative? `tensor` seems to be a misnomer after this PR

Reply:
Done, thanks